nodeinstaller/internal/constants/constants.go

```diff
@@ -56,7 +56,12 @@ func KataRuntimeConfig(baseDir string, platform platforms.Platform, qemuExtraKer
 		config.Hypervisor["clh"]["igvm"] = filepath.Join(baseDir, "share", "kata-containers-igvm.img")
 		config.Hypervisor["clh"]["image"] = filepath.Join(baseDir, "share", "kata-containers.img")
 		config.Hypervisor["clh"]["valid_hypervisor_paths"] = []string{filepath.Join(baseDir, "bin", "cloud-hypervisor-snp")}
-		config.Hypervisor["clh"]["enable_debug"] = debug
+		if debug {
+			config.Hypervisor["clh"]["enable_debug"] = true
+			config.Agent["kata"]["enable_debug"] = true
+			config.Agent["kata"]["debug_console_enabled"] = true
+			config.Runtime["enable_debug"] = true
```

Comment:
Should we move the non-hypervisor-specific (or maybe even the hypervisor-specific) parts out of the individual switch cases?